src/main/java/uk/ac/ebi/intact/service/complex/ws/SearchController.java

```diff
@@ -39,16 +26,14 @@
 import uk.ac.ebi.intact.service.complex.ws.model.ComplexRestResult;
 
 import javax.servlet.http.HttpServletResponse;
-import java.io.IOException;
-import java.io.StringWriter;
-import java.util.ArrayList;
 import java.util.List;
-import java.util.Map;
+import java.util.concurrent.CompletableFuture;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 @Log4j
 @Controller
+@AllArgsConstructor
 public class SearchController {
```

Comment:
Large diff in this class, but it's mostly cleaning up and moving stuff to other classes. Now the controller is mostly jus the endpoints.